### PR TITLE
add documentation on the before and after methods

### PR DESCRIPTION
**Pull request by agallou**

### CODE REVIEW

File: source/en/asserters/mock.inc.rst [hunk 1]
```diff
@@ -299,3 +299,67 @@ wasNotCalled
...
+           $mock->test2(),
+           $mock->test()
+       )
+       ->mock($mock)
+       ->call('test')
+           ->before($this->mock($mock)->call('test2')->once())
+           ->once() // fails
+   ;
+
+afer
```
Comment (marmotz) [inline source/en/asserters/mock.inc.rst]: afTer

File: source/fr/assertions/mock.inc.rst [hunk 2]
```diff
@@ -299,3 +299,67 @@ wasNotCalled
...
+           $mock->test2(),
+           $mock->test()
+       )
+       ->mock($mock)
+       ->call('test')
+           ->before($this->mock($mock)->call('test2')->once())
+           ->once() // échoue
+   ;
+
+afer
```
Comment (marmotz) [inline source/fr/assertions/mock.inc.rst]: afTer

**Comment by agallou** [comment]
@marmotz typo fixed

**Comment by marmotz** [comment]
Thank you !

**Pull request merged**